pkg/vault/test/foundry/BalancerPoolTokenTest.t.sol

```diff
@@ -134,7 +131,7 @@ contract BalancerPoolTokenTest is BaseVaultTest {
         assertEq(poolToken.nonces(user), CURRENT_NONCE + 1, "nonce mismatch");
     }
 
-    // @dev Just test for general fail as it is hard to compute error arguments
+    /// @dev Just test for general fail as it is hard to compute error arguments
```

Comment:
```suggestion
    /// @dev Just test for general fail as it is hard to compute error arguments.
```

If I'm not mistaken, `@notice` and `@dev` have periods at the end, and `@param` and `@return` don't, is that right?

Reply:
Yes; that's right.